src/plugins/gui/amountQuotePlugin.ts

```diff
@@ -157,15 +157,24 @@ export const amountQuoteFiatPlugin: FiatPluginFactory = async (params: FiatPlugi
         },
         label1: sprintf(lstrings.fiat_plugin_amount_currencycode, displayFiatCurrencyCode),
         label2: sprintf(lstrings.fiat_plugin_amount_currencycode, currencyCode),
-        onFieldChange: async (sourceFieldNum, value, stateManager): Promise<string | undefined> => {
+        onFieldChange: async (sourceFieldNum, value, stateManager) => {
```

Comment:
Still not sure I like this convention as it makes it "optional" to update the other field.